wallet-new/test/Integration/Expectations.hs

```diff
@@ -0,0 +1,29 @@
+module Integration.Expectations (MonadIntegrationExpectations(..)) where
+
+import           Universum
+
+import           Cardano.Wallet.Client.Http
+
+import           Integration.Util (Resp', asksM, fromResp)
+
+import qualified Test.Hspec as Hspec
+
+
+class MonadIntegrationExpectations m where
+    addressShouldExists :: Resp' WalletAddress -> m ()
+
+instance MonadIntegrationExpectations (ReaderT (WalletClient IO) IO) where
+    addressShouldExists resp = do
+        addr <- fromResp resp
+        addrs <- asksM getAddressIndex >>= fromResp
+        [ addrId | WalletAddress{..} <- addrs] `shouldContain` [addrId addr]
+
+
+--
+-- INTERNALS
+--
+
+-- | lifted 'Hspec.shouldContain'
+shouldContain :: (HasCallStack, Show a, Eq a) => [a] -> [a] -> ReaderT r IO ()
+shouldContain xs =
+    lift . Hspec.shouldContain xs
```

Comment:
Why not use [`hspec-expectations-lifted`](http://hackage.haskell.org/package/hspec-expectations-lifted)?

Reply:
:o !! That's EXACTLY what I needed here!